readme.md

```diff
@@ -1,41 +1,43 @@
 [`dotnet new` templates](https://github.com/dotnet/templating/wiki/%22Runnable-Project%22-Templates) for NServiceBus.
 
 
-## Docuemntation
+## Documentation
 
- * [Windows Service endpoint host](https://docs.particular.net/nservicebus/hosting/windows-service-template)
- * [ServiceControl Transport Adapter host](https://docs.particular.net/servicecontrol/transport-adapter/template)
+ * [NServiceBus Endpoint Windows Service](https://docs.particular.net/nservicebus/dotnet-templates#nservicebus-endpoint-windows-service)
+ * [ServiceControl Transport Adapter](https://docs.particular.net/nservicebus/dotnet-templates#servicecontrol-transport-adapter)
 
 
 ## Test locally
 
-After build templates can be installed using the following:
+After building the project, the templates can be installed using the following:
 
 
+### Re-Install
+
+```
+dotnet new -u ParticularTemplates
+dotnet new -i <path to project>\nugets\ParticularTemplates.xxx.nupkg
```

Comment:
`ParticularTemplates.xxx.nupkg` is that right? xyz?

Reply:
That's intended to be a placeholder for the version number on the package.